src/Xamarin.Android.Tools.ApiXmlAdjuster/JavaApi.XmlModel.cs

```diff
@@ -10,29 +10,77 @@ public partial class JavaApi
 	{
 		public JavaApi ()
 		{
-			Packages = new List<JavaPackage> ();
+			Packages = new Dictionary<string, JavaPackage> (StringComparer.OrdinalIgnoreCase);
 		}
 
 		public  string?                 ExtendedApiSource   { get; set; }
 		public  string?                 Platform            { get; set; }
-		public  IList<JavaPackage>      Packages            { get; set; }
+		public  IDictionary<string, JavaPackage>      Packages            { get; }
+
+		public ICollection<JavaPackage> AllPackages => Packages.Values;
 	}
 
 	public partial class JavaPackage
 	{
+		private Dictionary<string, List<JavaType>> types = new Dictionary<string, List<JavaType>> (StringComparer.OrdinalIgnoreCase);
```

Comment:
Java type names are not filesystem-bound constructs; they *are* case sensitive, and it is entirely valid to have a `package example` and `package Example` in the same app.  (It'll cause pain & suffering on case-insensitive filesystems!  But Java allows it.)

I think this should use `StringComparer.Ordinal`.

Reply:
Gode point, fixed.